x-pack/plugins/lens/public/heatmap_visualization/visualization.tsx

```diff
@@ -105,7 +105,7 @@ export const getHeatmapVisualization = ({
       label: i18n.translate('xpack.lens.heatmapVisualization.heatmapLabel', {
         defaultMessage: 'Heatmap',
       }),
-      groupLabel: groupLabelForBar,
+      groupLabel: groupLabelForHeatmap,
```

Comment:
Would it be possible to move the `Magnitude` group to appear before the `Proportion` group to better match the designs?